ovn_k8s/modes/overlay.py

```diff
@@ -178,12 +189,20 @@ def create_logical_port(self, event):
             return
 
         (mac_address, ip_address) = addresses.split()
-
         namespace = data['metadata']['namespace']
         pod_name = data['metadata']['name']
 
-        ip_address_mask = "%s/%s" % (ip_address, mask)
+        if create_network_policies:
```

Comment:
Do you think the following comment here makes sense (if it does not, would you mind writing the correct one):

When a pod gets created, we need to do two things.
1. Look at all network policies and see whether that network policy's pod selector matches this pod. If it does, add ingress control ACLs based on the rules in that policy object.

2. Look at all the network policies and see whether this pod matches any network policy's 'from' clause. If it does match, then we need to add this pod IP to that address set.